### PR TITLE
SDI-181: 🔧 Use new cimg redis executor

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

**Review by lbennett-moj — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,7 +29,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          command: 'sudo npm install -g npm@latest'
+          command: 'npm install -g npm@latest'
```
Comment (petergphillips) [inline .circleci/config.yml:32]: `16.15` doesn't contain `sudo` so removed

File: .dockerignore [hunk 2]
```diff
@@ -4,10 +4,9 @@ helm_deploy*
 README.md
 node_modules
 npm-debug.log
-README.md
```
Comment (petergphillips) [inline .dockerignore:7]: duplicate of line 4

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**